### PR TITLE
Correct incompatibleChannel log message

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: src/log.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export namespace message {
 
   export function incompatibleChannel(channel: Channel, markOrFacet: Mark | 'facet', when?: string) {
     return `${channel} dropped as it is incompatible with ${markOrFacet}` +
-      when ? `when ${when}` : '';
+      (when ? `when ${when}` : '')  + '.';
```
Comment (domoritz) [inline src/log.ts]: I think this misses a space. I will fix it differently.

**Comment by domoritz** [comment]
Check out my improvements and merge if you like them.

File: src/log.ts [hunk 2]
```diff
@@ -121,8 +121,7 @@ export namespace message {
   }
 
   export function incompatibleChannel(channel: Channel, markOrFacet: Mark | 'facet', when?: string) {
-    return `${channel} dropped as it is incompatible with ${markOrFacet}` +
-      (when ? `when ${when}` : '')  + '.';
+    return `${channel} dropped as it is incompatible with ${markOrFacet}${when ? ` when ${when}` : ''}.`;
```
Comment (kanitw) [inline src/log.ts]: I don't like nested string literals

Comment (kanitw) [inline src/log.ts]: But it's not a big deal. I'll just merge!

**Pull request merged**